contextproject-ms3/src/main/java/contextproject/models/BeatRange.java

```diff
@@ -0,0 +1,30 @@
+package contextproject.models;
+
+public class BeatRange {
+  public int start;
+  public int length;
+
+  /**
+   * Constructor for a beat range.
+   * 
+   * @param start
+   *          int start beat
+   * @param length
+   *          int range length
+   */
+  public BeatRange(int start, int length) {
+    this.start = start;
+    this.length = length;
+  }
+
+  @Override
+  public boolean equals(Object other) {
+    if (other instanceof BeatRange) {
+      BeatRange other2 = (BeatRange) other;
```

Comment:
Why make another beat? Can't you just make casts in the if case on "other"  instead of making another beatrange "other2"?